opentelemetry-sdk/tests/trace/export/test_export.py

```diff
@@ -482,7 +482,8 @@ def test_batch_span_processor_scheduled_delay(self):
         self.assertTrue(export_event.wait(2))
         export_time = time.time()
         self.assertEqual(len(spans_names_list), 1)
-        self.assertGreaterEqual((export_time - start_time) * 1e3, 500)
+        # call round to avoid flaky tests on pypy with delays ending some microseconds earlier
```

Comment:
If this is a `pypy`-specific problem, we should actually skip the test case for `pypy`, or mark it as flaky.

Reply:
It looks like the version of pypy we are using in ci has some timer precision issues on windows. This issues could have been fixed in 7.3.12 but that release has a baseline of python 3.9.

Reply:
> It looks like the version of pypy we are using in ci has some timer precision issues on windows. This issues could have been fixed in 7.3.12 but that release has a baseline of python 3.9.

Even more reason to skip it if it is being run with pypy in windows, then.